.github/workflows/build-then-deploy-docker.yml

```diff
@@ -0,0 +1,70 @@
+# Copyright 2023 Democratized Data Foundation
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.txt.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0, included in the file
+# licenses/APL.txt.
+
+# This workflow builds a Docker container image, if the build is successful
+# then it will deploy the image to DockerHub & GitHub container registries.
+name: Build Then Deploy Docker Workflow
+
+on:
+  pull_request:
```

Comment:
Issue: My hunch is that because you are using the fork-flow to test the trigger, `pull_request` has no access to secrets.